docs-src/python/how-to-set-asynchronous-activity-completion-in-python.md

```diff
@@ -0,0 +1,29 @@
+---
+id: how-to-set-asynchronous-activity-completion-in-python
+title: How to set an Asynchronous Activity Completion in Python
+sidebar_label: Set an Asynchronous Activity Completion
+description: To set the completion status of an asynchronous Activity, get the handle of the Activity and call the appropriate method of `AsyncActivityHandle`.
+tags:
+  - python
+  - how-to
+---
+
+To mark an Activity as completing asynchoronus, do the following.
+
+```python
+# Capture token for later completion
+captured_token = activity.info().task_token
+activity.raise_complete_async()
+```
+
+To update an Activity outside the Activity, use [AsyncActivityHandle](https://python.temporal.io/temporalio.client.AsyncActivityHandle.html) to get the handle of the Activity.
```

Comment:
`AsyncActivityHandle` _is_ the handle, you don't use it to get the handle. You use `get_async_activity_handle` to do that.